addons/default/visiosoft/base-theme/resources/config/settings/settings.php

```diff
@@ -79,7 +79,7 @@
 		'required' => false,
 		'config' => [
 			'default_value' => function () {
-				return [config('visiosoft.theme.base::countries.default')];
+				return config('visiosoft.theme.base::countries.default');
```

Comment:
is it working if it's empty? @spektra2147

Reply:
yes